apps/deploy-web/src/components/alerts/AlertsListView/AlertsListView.tsx

```diff
@@ -0,0 +1,224 @@
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow
+} from "@akashnetwork/ui/components";
+import { usePopup } from "@akashnetwork/ui/context";
+import { cn } from "@akashnetwork/ui/utils";
+import { Chip } from "@mui/material";
+import { createColumnHelper, flexRender, getCoreRowModel, useReactTable } from "@tanstack/react-table";
+import { Bin, Check, Edit } from "iconoir-react";
+import { capitalize, startCase } from "lodash";
+import Link from "next/link";
+
+import { UrlService } from "@src/utils/urlUtils";
+
+type Alert = components["schemas"]["AlertOutputResponse"]["data"];
+type AlertsInput = components["schemas"]["AlertListOutputResponse"]["data"];
+type AlertsPagination = components["schemas"]["AlertListOutputResponse"]["pagination"];
+
+export type AlertsListViewProps = {
+  data: AlertsInput;
+  pagination: Pick<AlertsPagination, "page" | "limit" | "total" | "totalPages">;
+  isLoading: boolean;
+  removingIds: Set<Alert["id"]>;
+  onRemove: (id: Alert["id"]) => Promise<void>;
+  onPaginationChange: (state: { page: number; limit: number }) => void;
+  isError: boolean;
+};
+
+export const AlertsListView: FC<AlertsListViewProps> = ({ data, pagination, onPaginationChange, isLoading, removingIds, onRemove, isError }) => {
+  const { confirm } = usePopup();
+  const columnHelper = createColumnHelper<Alert>();
+
+  const columns = [
+    columnHelper.accessor("name", {
+      header: () => <div>Name</div>,
+      cell: info => <div>{info.getValue()}</div>
+    }),
+    columnHelper.accessor("type", {
+      header: () => <div>Type</div>,
+      cell: info => <div>{startCase(info.getValue().toLowerCase())}</div>
+    }),
+    columnHelper.accessor("status", {
+      header: () => <div>Status</div>,
+      cell: info => <Chip color={info.getValue() === "NORMAL" ? "success" : "error"} label={capitalize(info.getValue())} />
+    }),
+    columnHelper.accessor("params", {
+      header: () => <div className="w-32">DSEQ</div>,
+      cell: info => {
+        const params = info.getValue();
+
+        return params ? (
+          <div className="flex max-w-48 items-center gap-1">
+            <Link href={UrlService.deploymentDetails(params.dseq, "ALERTS")} className="truncate">
+              {params.dseq}
+            </Link>
+          </div>
+        ) : (
+          <div className="text-gray-500">No parameters</div>
+        );
+      }
+    }),
+    columnHelper.accessor("enabled", {
+      header: () => <div>Enabled</div>,
+      cell: info => {
+        const enabled = info.getValue();
+
+        if (enabled) {
+          return <Check data-testid="alert-enabled-checkmark" className="text-sm text-green-600" />;
+        }
+      }
+    }),
+    columnHelper.display({
+      id: "actions",
+      cell: info => {
+        const isRemoving = removingIds.has(info.row.original.id);
+
+        return (
+          <div className="flex items-center justify-end gap-1">
+            <CustomTooltip title="Edit" disabled={isRemoving}>
```

Comment:
no need for the dedicated edit page for the initial iteration. Let's remove this for now and the pages too